ext/rugged/rugged_patch.c

```diff
@@ -344,6 +353,18 @@ static int patch_print_cb(
 
 	rb_ary_push(rb_buffer, rb_str_new(line->content, line->content_len));
 
+	if (print_payload->timeout > 0) {
+		struct timeval finish;
+		gettimeofday(&finish, NULL);
```

Comment:
@ethomson oh cool ok, let's just run with that then :)

@mclark we can pair on this again tomorrow if you'd like?